scripts/fstrim.sh

```diff
@@ -11,13 +11,11 @@ fi
 # Enter namespaces and run commands
 nsrun() {
     nsenter \
-        --mount="/host/proc/${KUBELET_PID}/ns/mnt" \
```

Comment:
The host path is still needed since the volume is mounted there

https://github.com/onedr0p/home-ops/blob/main/kubernetes/main/apps/kube-system/fstrim/app/helmrelease.yaml#L73